riff-raff/app/persistence/ContinuousDeploymentConfigRepository.scala

```diff
@@ -0,0 +1,40 @@
+package persistence
+
+import java.util.UUID
+
+import cats.data.Validated
+import ci.{ContinuousDeploymentConfig, Trigger}
+import com.gu.scanamo.{DynamoFormat, Scanamo, TypeCoercionError}
+import com.gu.scanamo.syntax._
+import conf.Configuration
+import org.joda.time.DateTime
+
+object ContinuousDeploymentConfigRepository {
+  val tableName = "continuous-deployment-config"
+  val client = Configuration.continuousDeployment.dynamoClient
```

Comment:
I have mixed feelings and the the ScanamoFree interface is designed to suit that model. 

In this case, since I'm directly replacing an existing interface, this was by far the path of least resistance.